dags/dts_work_zone_data_feed.py

```diff
@@ -0,0 +1,106 @@
+# test locally with: docker compose run --rm airflow-cli dags test dts_work_zone_data_feed
+
+import os
+
+from airflow.decorators import task
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+from pendulum import datetime, duration, now
+
+from utils.onepassword import get_env_vars_task
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "execution_timeout": duration(minutes=10),
```

Comment:
Just double-checking: this is enough time?

Reply:
I think so, usually takes me less than a minute to run. Also it'll be running hourly so we probably should put a cap on things to monitor it.